### ISSUE TITLE
Backup LED Panels config and restore

### ISSUE THREAD

**Issue by cybercop23** [FalconChristmas/fpp#1576]
**NOTE:** IF YOU DO NOT ATTACH YOUR LOG FILES, THEN THERE IS A HIGHER PROBABILITY THE ISSUE WON'T BE RESOLVED.  (Steps at end) 
# Please answer the following:
FPP 7.1 - any FPP vet

**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
When backing up config, either full or just led panels, the Width x Height  and Staarting location of the Panel config tab is not backed up, thus not being able to restored. Additionally, the layout is backed up but also not restored.

**Expected behavior**: 
Backup function should include the additional config attributes and be able to restore them via the restore option.

**Additional context**: 

**Additional Attachments**
See backed up config for this setup...

![image](https://github.com/FalconChristmas/fpp/assets/23619433/e0a08c0a-8d80-48e4-966d-66e01afd83aa)

[FPPPorch_channelOutputs-backup_v7_20230911175003.txt](https://github.com/FalconChristmas/fpp/files/12587866/FPPPorch_channelOutputs-backup_v7_20230911175003.txt)

**Comment by jaredb7** [comment]
I think I've fixed this. 
The panel start corner was restoring correctly but the Panel Layout WxH and Single Panel Size are being stored in the system settings and depending what was backed up and what was chosen to store those settings may be missing or not restore correctly.

When restore the channel outputs data or just the led panel config specifically, we cherry pick the Panel Layout setting  (from the system settings area) from the uploaded data and write that setting back to the system config. The Single Panel size was being missed completely though.

Thankfully they can be worked out the from panel data and existing backups will restore correctly now.

Your FPPPorch config supplied was a 3x4, but I just confirmed when restoring that was overwriting whatever random layout I set. And confirmed the layout looked the same.
